feature-group-definitions/messageerror.yml

```diff
@@ -0,0 +1,22 @@
+name: messageerror
+description: The `messageerror` event TODO.
+spec:
+  - https://html.spec.whatwg.org/multipage/indices.html#event-messageerror
+  - https://w3c.github.io/ServiceWorker/#eventdef-serviceworkerglobalscope-messageerror
+compat_features:
+  # BroadcastChannel as a whole wasn't supported in Edge 18.
+  #- api.BroadcastChannel.messageerror_event
```

Comment:
Is this exception needed? AFAICT, this key is younger than all the reported status versions, across the board.

Reply:
Including it changes Edge 18 to Edge 79, and that doesn't seem right for what is the intersection between two features.